src/owr/OptimisticWithdrawalRecipientPectra.sol

```diff
@@ -181,6 +194,19 @@ contract OptimisticWithdrawalRecipientPectra is Clone, OwnableRoles {
   /*     emit ReceiveETH(msg.value); */
   /* } */
 
+  function requestConsolidation(bytes memory data) external payable onlyOwnerOrRoles(CONSOLIDATION_WITHDRAWAL_ROLE) {
+    // ;; Input data has the following layout:
+    // ;;
+    // ;;  +--------+--------+
+    // ;;  | source | target |
+    // ;;  +--------+--------+
+    // ;;      48       48
+
+    (bool ret,) = pectraConsolidationAddress().call{value: msg.value}(data);
```

Comment:
we should make the pectra consolidation and withrawal address immutables/ constants via constructor

Reply:
Made them immutable